src/main/update_installer.js

```diff
@@ -0,0 +1,40 @@
+import {dialog, shell} from "electron";
+import phin from "phin";
+const semverGreaterThan = require("semver/functions/gt");
+const {version} = require("../../package.json");
+
+const getJSON = phin.defaults({method: "GET", parse: "json", headers: {"User-Agent": "BetterDiscord Installer"}});
+
+export default async function () {
+    const downloadUrl = "https://api.github.com/repos/BetterDiscord/Installer/releases";
+    console.info(`Better Discord Installer ${version}`);
+
+    try {
+        const response = await getJSON(downloadUrl);
+        const latestRelease = response.body[0];
+        const latestVersion = latestRelease.tag_name;
+
+        if (semverGreaterThan(latestVersion, version)) {
+            console.info(`Found new release ${latestVersion}`);
+
+            const result = await dialog.showMessageBox({
+                title: "New Installer Version Available",
+                message: `A new version of the BetterDiscord installer is available. Click "Download" to download the newest version.`,
+                buttons: ["Download", "Later"],
+                defaultId: 0,
+                cancelId: 1
+            });
+
+            if (result.response === 0) {
+                await shell.openExternal(latestRelease.html_url);
```

Comment:
Just a thought, but should we take them to the release page, or should we go to the corresponding OS's asset download url?

Reply:
I'm personally fine with either, the added complexity is minimal. but that would also require hard-coding (or at least semi-hardcoding) the filenames for the OS files.

Reply:
Well it wouldn't have to be complete hardcoded could do something like:
```js
let asset = assets.find(a => a.name.toLowerCase().includes("linux"); // Linux default 
if (process.platform === "win32") asset = assets.find(a => a.name.toLowerCase().includes("windows");
else if (process.platform === "darwin") asset = assets.find(a => a.name.toLowerCase().includes("mac");

await shell.openExternal(asset.browser_download_url);
```

But I'm still not sure if that option is truly better.

Reply:
An added bonus of linking the release page would beeing shown a changelog as well, even though though I expect most people to not read it.
And maybe we can put little text snippet on there after new releases that smart screen migtht pop up but it's  fine or something.

Reply:
I'm fine with that